### DISCUSSION TITLE
Outline view message for a .SQLRPGLE member

### DISCUSSION THREAD

**Discussion by hugocantor** [codefori/vscode-ibmi#557] [Q&A]
Hi,
  I'm very excited that Outline is available for RPG. I tried it with an sqlrpgle source member but unfortunately I'm getting a "No symbols found in document".  Please, I'd welcome any advise to get the Outline working, thanks! 

-Hugo

**Comment by worksofliam** [comment]
Hi @hugocantor!

When you have SQLRPGLE open, what language ID does it use? Can you see the language ID in the bottom right-hand corner? It should say 'RPGLE':

<img width="1168" alt="image" src="https://user-images.githubusercontent.com/3708366/155189028-5a6fe43c-4047-47c7-86c6-f44d4337b567.png">

**Reply by hugocantor** [reply]
@worksofliam , Hi Barry, Yes, it does display "RPGLE", but I still get the same message. To get to the Outline view I'm going to View, Open View, Outline.

**Reply by worksofliam** [reply]
@hugocantor Do you also have RPGLE language tools installed? https://marketplace.visualstudio.com/items?itemName=HalcyonTechLtd.vscode-rpgle

**Reply by hugocantor** [reply]
@worksofliam , yes

<img width="960" alt="rpgle_languaje_tools_installed" src="https://user-images.githubusercontent.com/29375705/155196719-29712ced-5be9-4858-9feb-a0718ca8b45a.png">

**Reply by worksofliam** [reply]
@huge If the outline doesn't work in this case, you will need to share the code that it doesn't work with and I can take a look to see if it's a deeper issue. Thanks!

**Reply by worksofliam** [reply]
Issue was found with RPGLE language tools. Fix and release will be out tonight.

**Reply by hugocantor** [reply]
Thank you!
